packages/turbo-ignore/src/index.ts

```diff
@@ -7,6 +7,15 @@ import { getComparison } from "./getComparison";
 console.log(
   "\u226B Using Turborepo to determine if this project is affected by the commit..."
 );
+
+// check for TURBO_FORCE and bail early if it's set
+if (process.env.TURBO_FORCE === "true") {
+  console.log(
+    "\u226B Turbo force detected, skipping check and proceeding with build."
```

Comment:
```suggestion
    "\u226B `turbo --force` detected, skipping check and proceeding with build."
```

Reply:
Good call, going to use `TURBO_FORCE` just for consistency with where we're reading this from.